Dashboard/App.axaml.cs

```diff
@@ -27,12 +27,19 @@ public override void OnFrameworkInitializationCompleted()
             // Setup dependency injection
             var services = DependencyInjection.ConfigureServices();
 
-            desktop.MainWindow = new MainWindow
+            /*desktop.MainWindow = new MainWindow
```

Comment:
I know that these windows won't be displayed like this in the future, but for the purpose of this ticket you can just replace this line with
```suggestion
            desktop.MainWindow = new ConnectionWindow();
```